napari/components/_viewer_key_bindings.py

```diff
@@ -121,6 +121,10 @@ def toggle_selected_visibility(viewer: Viewer):
     viewer.layers.toggle_selected_visibility()
 
 
-@register_viewer_action(trans._("Show/Hide IPython console"))
+@register_viewer_action(
+    trans._(
+        "Show/Hide IPython console (only available when napari started as standalone application)"
+    )
+)
```

Comment:
maybe part in `()` should be visible only if napari is run from ipython? The end of the message is obsolete for bundle app users and may reduce readability.